_posts/2019-12-03-accenture-ctf-2019.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: post
+title: DISsect@CTU 3rd at Accenture Ctf 2019
+date: 2019-12-03 13:45:56
+author: ordbl0ke
+published: true
+tags: [ctf, summary, accenture]
+---
+
+#### CTF Info
+
+__url__: https://ctf.accenture.com \\
+__date:__ `14/11/2019` (12 hours)
+
+On 14th of November 2019, we took part in [ctf](https://ctf.accenture.com) competition organised by Accenture. Team consisting of `ivanivlee`, `h4vry`, `TheLonePiper` and `ordbl0ke` rose to the occasion and finished **third** in the late dramatic finale! 
+
+You can find out more about the competition [here](https://ctf.accenture.com).
+
+<!--more-->
+
+#### More
+
+The competition required the contestants to be students and limited the team size to 4. This was unfortunate as **DISsecT** has more than 4 members and we had to split the team into two. Team of `ivanivlee`, `h4vry`, `TheLonePiper` and `ordbl0ke` benefited from having 4 participants and succesfully grabbed a podium spot. Team that contested from a different, remote location to prevent inter-team cooperation, combined the efforts of `gorei` and `hrdel`. They fell short to the numbers, but ended up on a solid, 11th spot, leaving more than two thirds of the contesting teams behind.
```

Comment:
```suggestion
The competition required the contestants to be students and limited the team size to 4. This was unfortunate as **DISsecT** has more than 4 members and we had to split the team into two. Team of `ivanivlee`, `h4vry`, `TheLonePiper` and `ordbl0ke` benefited from having 4 participants and successfully grabbed a podium spot. The team that contested from a different, remote location to prevent inter-team cooperation, combined the efforts of `gorei` and `hrdel`. They fell short to the numbers, but ended up on a solid, 11th spot, leaving more than two-thirds of the contesting teams behind.
```